types/src/crypto/ratchet/snapshots/phnxtypes__crypto__ratchet__tests__ratchet_secret_serde_stability_json.snap

```diff
@@ -0,0 +1,85 @@
+---
+source: types/src/crypto/ratchet/tests.rs
+expression: "&*RATCHET"
+snapshot_kind: text
+---
+{
+  "sequence_number": 0,
+  "secret": {
+    "key": {
+      "secret": [
+        152,
+        25,
+        31,
+        70,
+        229,
+        131,
+        2,
+        22,
+        68,
+        84,
+        54,
+        151,
+        136,
+        3,
+        105,
+        122,
+        94,
+        58,
+        182,
+        27,
+        30,
+        137,
+        81,
+        212,
+        254,
+        154,
+        230,
+        123,
+        171,
+        97,
+        74,
+        95
+      ]
+    }
+  },
+  "key": {
+    "key": {
+      "secret": [
+        74,
+        56,
+        89,
+        181,
+        167,
+        124,
+        19,
+        57,
+        122,
+        40,
+        120,
+        59,
+        56,
+        173,
+        111,
+        215,
+        122,
+        210,
+        102,
+        143,
+        71,
+        2,
+        149,
+        13,
+        217,
+        68,
+        161,
+        31,
+        82,
+        43,
+        131,
+        164
+      ]
+    }
+  },
+  "_phantom": null
```

Comment:
Looks like we have now this in our format. Removing it breaks both JSON and CBOR.